### PR TITLE
Musikhin Vlad 203

### PR DESCRIPTION
**Pull request by UncleTm-07**

### CODE REVIEW

File: item.html [hunk 1]
```diff
@@ -2,9 +2,85 @@
...
+    <div class="header_menu" id="backMainPage">
+      <img src="/img/item/back.svg" alt="">
+    </div>
+    <div class="header_title">
+      <h3 class="title">Тур вихідного дня “Чор...</h3>
+    </div>
+  </header>
+  <main class="main">
+    <div class="card">
+      <img src="img/1.png" alt="">
```
Comment (Zadvornyi) [inline item.html]: Please use relative path for avoiding problem this probkem

Comment (Zadvornyi) [reply item.html]: Screenshot 2023-09-10 at 17 18 10

**Review by Zadvornyi — APPROVED** [review]

**Comment by UncleTm-07** [comment]
Fix was complited!

**Comment by UncleTm-07** [comment]
Please check my commit!!!
Path was fixed

**Pull request closed without merging**